association-frontend/pages/signup/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
```

Comment:
I am not a big fan of leaving unused code in PRs, if we don't have signup/login and so on, does it make sense to not merge it for now?